index.html

```diff
@@ -18,113 +18,80 @@
         <div class="container">
             <div class="card">
                 <div class="price-circle">
```

Comment:
```suggestion
        <div class="card__price">
          <h2>$399</h2>
          <span>Per tour</span>
        </div>
```
e để cấu trúc như này cho cả 3 card nha. 3 card này giống nhau nhưng a thấy card ở giữa e đang để 1 class khác.